rack-test.gemspec

```diff
@@ -1,25 +1,26 @@
 # -*- encoding: utf-8 -*-
+# stub: rack-test 0.6.3 ruby lib
```

Comment:
@voxik thank you for mentioning it. Yes, this PR includes the gemspec file generated from `Thorfile` by `thor`. The line will be removed by PR: https://github.com/rack-test/rack-test/pull/181 .